src/main/java/eu/stamp_project/testrunner/listener/junit4/JUnit4TestResult.java

```diff
@@ -49,38 +49,38 @@ public void testFinished(Description description) throws Exception {
 
     @Override
     public void testFailure(org.junit.runner.notification.Failure failure) throws Exception {
-        this.internalTestResult.getFailingTests().add(
-                new Failure(
-                        this.toString.apply(failure.getDescription()),
-                        failure.getDescription().getClassName(),
-                        failure.getException()
-                )
-        );
+    	this.internalTestResult.getFailingTests().add(
+    			new Failure(
+    					this.toString.apply(failure.getDescription()),
+					    failure.getDescription().getClassName(),
+					    failure.getException()
+			    )
+	    );
     }
 
     @Override
     public void testAssumptionFailure(org.junit.runner.notification.Failure failure) {
-        this.internalTestResult.getAssumptionFailingTests().add(
-                new Failure(
-                        this.toString.apply(failure.getDescription()),
+    	this.internalTestResult.getAssumptionFailingTests().add(
+    			new Failure(
+    					this.toString.apply(failure.getDescription()),
                         failure.getDescription().getClassName(),
                         failure.getException()
-                )
+			    )
         );
     }
 
     @Override
     public void testIgnored(Description description) throws Exception {
-        this.internalTestResult.getIgnoredTests().add(this.toString.apply(description));
+    	this.internalTestResult.getIgnoredTests().add(this.toString.apply(description));
```

Comment:
Could you please indent properly this line?

Reply:
Done!